Installation-CentOS-8-Apache

```diff
@@ -0,0 +1,365 @@
+source: Installation/Installation-CentOS-8-Apache.md
+path: blob/master/doc/
+
+> NOTE: These instructions assume you are the **root** user.  If you
+> are not, prepend `sudo` to the shell commands (the ones that aren't
+> at `mysql>` prompts) or temporarily become a user with root
+> privileges with `sudo -s` or `sudo -i`.
+
+**Please note the minimum supported PHP version is 7.2.5**
+
+**This installation guide will install the [Dispatcher Service](https://docs.librenms.org/Extensions/Dispatcher-Service/) for a standalone installation by default**
+
+## Install Common Required Packages ##
+
+```
+dnf install epel-release
+```
+Enable `PowerTools` repo to be able to install the `whois` package
+
+```
+dnf config-manager --set-enabled PowerTools
+dnf install python3 git cronie fping whois ImageMagick mtr python3-PyMySQL net-snmp net-snmp-utils nmap python3-memcached rrdtool python3-policycoreutils httpd mariadb mariadb-server unzip policycoreutils-python-utils
+pip3 install python-dotenv
+```
+To fix compatibility issues with hardcoded python paths, create a symlink for python3
+
+```
+ln -sf /usr/bin/python3 /usr/bin/python
+```
+
+### Install PHP
+
+CentOS 8 ships with php 5.x which is not compatible with LibreNMS.
+There are multiple ways to install php 7.x on CentOS 8, like Remi, SCL or Webtatic. The first two are maintained by Remi Collet of RedHat.
+
+#### Running with Remi PHP
+
+Set up the repo to be used
+```
+dnf install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+dnf install https://rpms.remirepo.net/enterprise/remi-release-8.rpm
+```
+
+Install the required packages
+
+```
+dnf module install php:remi-7.3
+dnf install mod_php php-gd php-process php-snmp php-pecl-zip php-pecl-memcached php-mysqlnd
+```
+
+# Add librenms user
+
+```
+useradd librenms -d /opt/librenms -M -r
+usermod -a -G librenms apache
+```
+
+# Download LibreNMS
+
+```
+cd /opt
+git clone https://github.com/librenms/librenms.git
+```
+
+# Set permissions
+
+```
+chown -R librenms:librenms /opt/librenms
+chmod 770 /opt/librenms
+setfacl -d -m g::rwx /opt/librenms/rrd /opt/librenms/logs /opt/librenms/bootstrap/cache/ /opt/librenms/storage/ /opt/librenms/cache
+setfacl -R -m g::rwx /opt/librenms/rrd /opt/librenms/logs /opt/librenms/bootstrap/cache/ /opt/librenms/storage/ /opt/librenms/cache
+```
+
+# Install PHP dependencies
+
+```
+su - librenms
+./scripts/composer_wrapper.php install --no-dev
```

Comment:
This seems out of order,  I try to do all the stuff that requires root first, then do stuff as the librenms user.